migrations/20191119151817-create-medical-condition-treatment-join-table.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('MedicalConditionTreatments', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      medical_condition_id: {
```

Comment:
Comment: Chatted with Kyle about whether there is ever a circumstance when a medication or treatment not associated with a condition. We're going with no (which is as the implementation does), but might be a thing worth changing later if research testing says otherwise.